instrumentation-genai/opentelemetry-instrumentation-openai-v2/README.rst

```diff
@@ -17,6 +17,53 @@ Installation
 
     pip install opentelemetry-instrumentation-openai-v2
 
+Usage
+-----
+
+Instrumenting all clients
+*************************
+
+When using the instrumentor, all clients will automatically trace OpenAI chat completion operations
+and capture prompts and completions as events.
+
+Make sure to configure OpenTelemetry tracing, logging, and events to capture all telemetry emitted by the instrumentation.
+
+.. code-block:: python
+
+    from opentelemetry.instrumentation.openai_v2 import OpenAIInstrumentor
+
+    OpenAIInstrumentor().instrument()
+
+    client = OpenAI()
+    response = client.chat.completions.create(
+        model="gpt-4o-mini",
+        messages=[
+            {"role": "user", "content": "Write a short poem on open telemetry."},
+        ],
+    )
+
+Enabling message content
```

Comment:
was this snippet about configuring logs and events meant to be in here, too? https://github.com/open-telemetry/opentelemetry.io/pull/5575#discussion_r1835188607

Reply:
added manual configuration example with this code snippet.